tests/test_tlsfuzzer_expect.py

```diff
@@ -1432,9 +1523,12 @@ def test_process_with_tls_1_2_wrong_downgrade_protection(self):
 
         self.assertTrue(exp.is_match(msg))
 
-        with self.assertRaises(AssertionError):
+        with self.assertRaises(AssertionError) as e:
             exp.process(state, msg)
 
+        self.assertIn("downgrade protection sentinel but shouldn",
```

Comment:
how does e get here ? shouldn't this be indented in the with statement above ?

Reply:
just as above - assertRaises catches the exception